common/src/primitives/id.rs

```diff
@@ -46,6 +53,10 @@ impl<T> Id<T> {
             _shadow: std::marker::PhantomData,
         }
     }
+
+    pub fn is_null(&self) -> bool {
```

Comment:
This has to go... certainly... but soon, when the coinbase thing is resolved.

Reply:
Ok.